src/node_internals.h

```diff
@@ -324,8 +324,9 @@ namespace credentials {
 bool SafeGetenv(const char* key, std::string* text, Environment* env = nullptr);
 }  // namespace credentials
 
-void PrintTraceEnvStack(Environment* env);
-void PrintTraceEnvStack(std::shared_ptr<EnvironmentOptions> options);
+void TraceEnv(Environment* env, const char* message);
```

Comment:
The abbreviated name reminds me that the term "Env" here does not refer to `node::Environment`, but "Environment variable" instead. Can we clarify this with a name like `TraceEnviron`?